folium/map.py

```diff
@@ -480,3 +480,46 @@ def __init__(self, bounds, padding_top_left=None,
             padding_bottom_right=padding_bottom_right,
             padding=padding,
         )
+
+
+class CustomPane(MacroElement):
+    """
+    Creates a custom pane to hold map elements.
+
+    Behavior is as in https://leafletjs.com/examples/map-panes/
+
+    Parameters
+    ----------
+    name: string
+        Name of the custom pane. Other map elements can be added
+        to the pane by specifying the 'pane' kwarg when constructing
+        them.
+    z_index: int or string, default 625
+        The z-index that will be associated with the pane, and will
+        determine which map elements lie over/under it. The default
+        (625) corresponds to between markers and tooltips. Default
+        panes and z-indexes can be found at
+        https://github.com/Leaflet/Leaflet/blob/v1.0.0/dist/leaflet.css#L87
```

Comment:
Could you link here to the Leaflet documentation instead? https://leafletjs.com/reference-1.4.0.html#map-pane
It has the same info, is easier to read, and using this URL makes it easy for us to update all links when a new Leaflet version is released.